openmetadata-spec/src/main/resources/json/schema/entity/services/connections/metadata/atlasConnection.json

```diff
@@ -39,22 +39,10 @@
       "description": "entity types of the data source.",
       "type": "string"
     },
-    "serviceType": {
-      "description": "service type of the data source.",
-      "type": "string"
-    },
     "atlasHost": {
       "description": "Atlas Host of the data source.",
       "type": "string"
     },
-    "dbService": {
-      "description": "source database of the data source.",
-      "type": "string"
-    },
-    "messagingService": {
-      "description": "messaging service source of the data source.",
-      "type": "string"
-    },
```

Comment:
can you explain a bit more here?

Reply:
So, we are using these to create a service of name `dbService` of type  `serviceType` and messaging Service of `messagingService`. So removing these properties. Make these an automatic process.